### ISSUE TITLE
Treatment of `target="_blank"`

### ISSUE THREAD

**Issue by back2dos** [kevinresol/coconut.router#4]
While using `target="_blank"` is arguably bad practice, I wonder if such links should really be intercepted.

**Comment by kevinresol** [comment]
Yeah I agree we should not intercept links that had non-self target
